pyvrp/Population.py

```diff
@@ -131,26 +126,32 @@ def select(self) -> Tuple[Individual, Individual]:
         tries = 1
         while not (lb <= diversity <= ub) and tries <= 10:
             tries += 1
-            second = self.get_binary_tournament()
+            second = self.get_binary_tournament(rng)
             diversity = self._op(first, second)
 
         return first, second
 
     def restart(self):
         """
-        Restarts the population. All individuals are removed and a new initial
-        population population is generated.
+        Restarts the population. All current individuals are removed and the
+        original initial individuals are used agains to initialize the
+        restarted population.
```

Comment:
Typo: agains.

Can we rewrite this a bit to really clarify that we're restarting from the initial individuals that were passed when this object was initialised? It's not clear enough to me what "original initial individuals" means.